tests/unit/lib/functions/registry.test.js

```diff
@@ -0,0 +1,33 @@
+const { mkdirSync, mkdtempSync, writeFileSync } = require('fs')
```

Comment:
[dust] Consider using the async version of these methods.

Reply:
I changed them out.  Some of the syntax is older as the import style changes between Node 12 and Node 14, but the CLI still supports Node 12.